OpenMcdf.Ole/DictionaryProperty.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace OpenMcdf.Ole;
 
-public class DictionaryProperty : IProperty
+public sealed class DictionaryProperty : IProperty
```

Comment:
TBD: I'm not sure if this needs to be public at all

Reply:
OLE isn't really my thing, so I have no idea how this intended to be used. I'm generally in favour of sealing unless there's a reason not to and the class is designed to be extended though. I'm happy to defer to your judgement. The rest all LGTM.

Reply:
Basically I'm not clear why DictionaryProperty would be public when all the other implementations of IProperty (i.e the TypedPropertyValue classes) appear to be internal.

Usage of those classes is generally part of PropertyFactory and/or PropertySetStream, and both of those are internal